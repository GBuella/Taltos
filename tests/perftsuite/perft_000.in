setboard rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1
echo rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
