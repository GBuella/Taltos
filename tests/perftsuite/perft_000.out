rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1
20
400
8902
197281
4865609
119060324
